Clarify TIFF/TIF images must be `uint8` (#22756)
ultralytics/ultralytics: docs/en/datasets/detect/coco8-multispectral.md

!!! note

>     Prepare your TIFF images in `(channel, height, width)` order and saved with `.tiff` or `.tif` extension for use with Ultralytics. The TIFF images must be `uint8`:

Reviewer: 📝 **LOW**: The added sentence reads awkwardly because the colon that introduces the code block is now attached to the “must be `uint8`” statement rather than the instruction as a whole. Consider merging the sentences so the colon clearly applies to the entire guidance.

**Suggested change:**
```suggestion
    Prepare your TIFF images in `(channel, height, width)` order, saved with `.tiff` or `.tif` extension, and ensure they are `uint8` for use with Ultralytics:
```
